gradle/libs.versions.toml

```diff
@@ -58,4 +58,5 @@ kotest-property = { module = "io.kotest:kotest-property-jvm", version.ref = "kot
 quickcheck = { module = "net.java:quickcheck", version.ref = "quickcheck" }
 
 [bundles]
-kotest = ["kotest-runner", "kotest-assertions", "kotest-property"]
\ No newline at end of file
+kotest = ["kotest-runner", "kotest-assertions", "kotest-property"]
+playservices = ["playservices-base", "playservices-basement", "playservices-tasks"]
```

Comment:
Given that common does depend on base and tasks, but not basements, I think there's a point to be made to not group them all together.

Reply:
Folks can still pull dependencies on individual libs, for example:
```toml
implementation(libs.bundles.playservices)
implementation(libs.playservices.base)
```

Additionally, `functions` could take advantage of this bundle- as they depend on all three as well.